types/docusign-esign/docusign-esign-tests.ts

```diff
@@ -34,6 +34,19 @@ const getEnvelope = async (
     return results;
 };
 
+const getEnvelopeWithStoredConfiguredClient = async (
+    envelopeId: string,
+    options: { advancedUpdate?: string | undefined; include?: string | undefined },
+) => {
+    const params = await getDsRequestParams();
+    const client = await getClient(params.token);
+    docusign.Configuration.default.setDefaultApiClient(client);
+    const envelopesApi = new docusign.EnvelopesApi();
```

Comment:
example here of setting the default apiClient and then getting an instance of the EnvelopesApi without passing an apiClient into the constructor.